### ISSUE TITLE
Very laggy

### ISSUE THREAD

**Issue by klaszlo8207** [hemanthrajv/flutter_3d_obj#13]
The viewer is very slow on a HIGH end machine

**Comment by klaszlo8207** [comment]
it is only laggy when you need to draw a lot of points

**Comment by hemanthrajv** [comment]
Yeah, it is because it's take lots of time time computing. Lots of points
put more work on the CPU.

**Comment by jpnurmi** [comment]
- consider parsing .obj once after reading, not in paint() after every rotation etc.
- and batching huge amounts of Canvas.drawPath() calls with help of Canvas.drawVertices()

**Comment by nabil-ha** [comment]
> 
> 
>     * consider parsing .obj once after reading, not in paint() after every rotation etc.
> 
>     * and batching huge amounts of Canvas.drawPath() calls with help of Canvas.drawVertices()

How to do that? can you explain?

**Comment by youssefali424** [comment]
what he means fork the package change the part starting line 307, in paint function he is reading the file imagine that when you change the rotation he reads the whole file again that's too much I will try the edits locally if it worked I will refer to the forked git here or perhaps a pr
